File: src/au/com/rmit/Game2dEngine/physics/gravity/Gravity.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package au.com.rmit.Game2dEngine.physics.gravity;

/**
 *
 * @author ricolwang
 */
public class Gravity
{

    public double GX = 9.8f;
    public double GY = 0f;

    public Gravity(double gx, double gy)
    {
        this.GX = gx;
        this.GY = gy;
    }

}

File: src/com/wang/Game2dEngine/physics/gravity/Gravity.kt
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.wang.Game2dEngine.physics.gravity

/**
 * @author ricolwang
 */
class Gravity
{
    var GX: Double
    var GY: Double

    constructor(gx: Double, gy: Double)
    {
        GX = gx
        GY = gy
    }

    constructor(gx: Int, gy: Int)
    {
        GX = gx.toDouble()
        GY = gy.toDouble()
    }
}